static/js/front.js

```diff
@@ -19,12 +19,13 @@ $(function () {
   animations()
   counters()
   demo()
-  contactForm()
+  contactFormAjax()

```

Comment:
I'm not sure if it's correct, I would avoid to call `contactFrom()` if it's not present the form with class `.contact-form-ajax`

Reply:
I agree with the rename `contactFormAjax()`

Reply:
Ok. I left the rename.